bool is_connected # xbox360 USB controller/dongle is connected
bool is_linked    # wireless xbox360 has [no] link with dongle
